src/site/includes/debug.drupal.liquid

```diff
@@ -1,4 +1,8 @@
-{% if buildtype != 'vagovprod' %}
+{% if isPreview or buildtype != 'vagovprod' and omitdebug = false %}
```

Comment:
This is the only successful format I could find for this. Liquid apparently has really bad boolean arithmetic. It's even required to set a variable like this, using the conditional inline doesn't work.

Reply:
Haha yeah, that is about what I expected about the parentheses. Thanks for trying though bc I was curious

Reply:
Would this work?

```suggestion
{% if omitdebug = false %}
  {% if isPreview or buildtype != 'vagovprod' %}
```

I like this better because it removes the ambiguity. With your change it's hard to tell how the logic is grouped.

Reply:
`omitdebug == false` was causing the issues in the first place, so we need the `isPreview` conditional first